frontend/public/components/monitoring/receiver-forms/alert-manager-receiver-forms.tsx

```diff
@@ -414,11 +414,11 @@ const ReceiverBaseForm: React.FC<ReceiverBaseFormProps> = ({
           </div>
           <div className="form-group">
             <label className="co-required">{t('public~Receiver type')}</label>
-            <Dropdown
+            <ConsoleSelect
```

Comment:
same issue on Alert receiver creation form page

https://github.com/user-attachments/assets/431899e3-d333-412b-b442-26f36da9e477

Reply:
going to revert this change when https://github.com/openshift/console/pull/15275 merges